### ISSUE TITLE
Add badges

### ISSUE THREAD

**Issue by greysteil** [dependabot/dependabot-core#374]
_From @sobolevn on November 17, 2017 9:16_

I want to inform my users, that dependencies are up-to-date.

The easiest way to do it, as for me, is a badge. Something similar to:

![dependencies](https://img.shields.io/david/expressjs/express.svg)
![dev](https://img.shields.io/david/dev/expressjs/express.svg)

Do you have any plans to support this feature?

_Copied from original issue: dependabot/feedback#49_

**Comment by greysteil** [comment]
Yes, we'd be really keen to support that. I want to wait a month to see how GitHub's dependency graph support shakes out (it might well make more sense to power a badge with that), but if there's a role for Dependabot to play here we'll do it. Would be free publicity for us, after all!

**Comment by greysteil** [comment]
_From @StephanBijzitter on November 20, 2017 12:58_

Don't forget to throw in a badge that is used only to advertise :-)
https://img.shields.io/badge/Dependabot-active-brightgreen.svg
![Example shield](https://img.shields.io/badge/Dependabot-active-brightgreen.svg)

**Comment by greysteil** [comment]
_From @mydearxym on March 21, 2018 3:42_

+1

**Comment by simlu** [comment]
@greysteil You should really update the repo description in that case. It currently says that this is the place for feedback :)

**Comment by axelrindle** [comment]
For an "advertising" badge one could use [badgen](https://badgen.net) for the moment, which also supports an icon.

[![Dependabot](https://badgen.net/badge/Dependabot/enabled/green?icon=dependabot)](https://dependabot.com/)
```
[![Dependabot](https://badgen.net/badge/Dependabot/enabled/green?icon=dependabot)](https://dependabot.com/)
```

**Comment by greysteil** [comment]
Thanks @axelrindle! There’s actually another issue for this on https://github.com/dependabot/feedback and we now have a marketing badge, and it does use Badgen! If you comment `@dependabot badge me` on a dependabot PR it will give you the right markdown 😎

**Comment by axelrindle** [comment]
@greysteil That's great! :+1:
